### PR TITLE
Added Data Prepper 1.3.0 archives

### PR DESCRIPTION
**Pull request by asifsmohammed**
Signed-off-by: Asif Sohail Mohammed <nsifmoh@amazon.com>

### Description
Added Data Prepper 1.3.0 linux archives, Dockerfile image and updated version to 1.3.0

This PR is for an upcoming release. Please wait for confirmation before merging.

 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+role: ingest
+artifact_id: data-prepper
+version: data-prepper-1.3.0
+platform: linux
+architecture: x64
+type: targz
+artifact_url: https://artifacts.opensearch.org/data-prepper/1.2.1/opensearch-data-prepper-jdk-1.3.0-linux-x64.tar.gz
```
Comment (dlvenable) [inline _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown:8]: The filename is correct using `1.3.0`, but you still have `1.2.1` in the path leading up to it. The directory structure does have the version in the name and we need to update accordingly.

File: _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+signature: https://artifacts.opensearch.org/data-prepper/1.2.1/opensearch-data-prepper-jdk-1.3.0-linux-x64.tar.gz.sig
```
Comment (dlvenable) [inline _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown:9]: Update version number in path. See comment above.

**Review by dlvenable — CHANGES_REQUESTED** [review]

File: _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown (hunk 1 above)
Comment (asifsmohammed) [reply _artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown:8]: I'll update it. Thanks!

**Review by dlvenable — DISMISSED** [review]

**Review by dlvenable — APPROVED** [review]

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**